tutorials/assets_pipeline/importing_3d_scenes/node_type_customization.rst

```diff
@@ -17,7 +17,8 @@ will detect suffixes in object names and will perform actions automatically.
 
 .. warning::
 
-    All the suffixes described below are **case-sensitive**.
+    All the suffixes described below can be used with ``$``, ``-`` and ``_`` and are 
```

Comment:
```suggestion
    All the suffixes described below can be used with ``$`` and ``_`` instead of ``-`` and are 
```
Or
```suggestion
    All the suffixes described below can be used with ``$``, ``-``, and ``_`` and are 
```
If original form is kept